### ISSUE TITLE
Add Spanish language support

### ISSUE THREAD

**Issue by mattyweb** [hackforla/311-data#1004]
### Dependency
This issue is dependent on user testing and surveying if Spanish language support is a priority feature for our target customers. 

### Description
We should have Spanish language support for the app. We could auto-detect from the browser setting, but then also let the user choose a preferred language.

### Action Items

Need to pick a library to do the translation. Need to create a translation file with the Spanish strings.

### Resources
https://www.i18next.com/overview/plugins-and-utils
https://lokalise.com/blog/how-to-internationalize-react-application-using-i18next/
https://medium.com/@jishnu61/6-easy-steps-to-localize-your-react-application-internationalization-with-i18next-8de9cc3a66a1
